financial-templates-lib/logger/SlackTransport.js

```diff
@@ -1,4 +1,4 @@
-// This transport enables slack messages to be sent from twillio logging. To configure this
+// This transport enables slack messages to be sent from Winston logging. To configure this
```

Comment:
Review for this file is in [PR](https://github.com/UMAprotocol/protocol/pull/1185)